### PR TITLE
Adding a wind average display next to the wind min and wind max. Also…

### PR DESCRIPTION
**Pull request by KyleAnthonyShepherd**
… added a wind risk percentage in the hover tooltip
image
image

### CODE REVIEW

**Comment by WatchTheFort** [comment]
I think this is a bit much, in terms of overcomplicating how wind is presented in the game

File: luaui/Widgets/gui_top_bar.lua [hunk 1]
```diff
@@ -655,7 +668,18 @@ local function updateWind()
 	end)
 
 	if WG['tooltip'] ~= nil then
-		WG['tooltip'].AddTooltip('wind', area, Spring.I18N('ui.topbar.wind.tooltip', { titleColor = textTitleColor, warnColor = textWarnColor }))
+		risk=riskWind[minWind]
+		if risk ~= nil then
+			risk=risk[maxWind]
+		end
+		if risk == nil then
+			if minWind+maxWind >= 0.5 then
```
Comment (sprunk) [inline luaui/Widgets/gui_top_bar.lua:676]: Where does the 0.5 come from?

Comment (KyleAnthonyShepherd) [reply luaui/Widgets/gui_top_bar.lua:676]: Other places in gui_top_bar.lua use "if minWind+maxWind >= 0.5 then" to special case no-wind maps. So I'm personally not sure why that method is used.

**Comment by KyleAnthonyShepherd** [comment]
Removed average wind number from wind UI box, and moved it to the hover tooltip

image

**Comment by badosu** [comment]
LGTM! :+1:

**Pull request merged**